### PR TITLE
Add the --nogpgcheck option

### PR DESCRIPTION
**Pull request by gaowanlong**
I got the deploy failure with local mirror and the official gpgkeys,
the error is like:

	[node-1][WARNIN]  Failing package is: 1:librbd1-0.94.6-0.el7.x86_64
	[node-1][WARNIN]  GPG Keys are configured as: file:///opt/ceph-deploy/repo/release.asc

Anyway, I do not care the gpg keys but want to install successfully
without gpgcheck.

Signed-off-by: Wanlong Gao <wanlong.gao@easystack.cn>

### CODE REVIEW

File: ceph_deploy/install.py [hunk 1]
```diff
@@ -584,6 +589,12 @@ def make(parser):
                 (defaults to ceph.com)'
     )
 
+    repo.add_argument(
+        '--nogpgcheck',
```
Comment (alfredodeza) [inline ceph_deploy/install.py]: this would not be necessary if the repo is configured with `gpgcheck=0`, right?

Comment (gaowanlong) [inline ceph_deploy/install.py]: @alfredodeza Yeah, it is useful when the "--local-mirror" or "--repo-url" is specified to leave the ceph-deploy do other repo configurations.

Thanks,
Wanlong

Comment (gaowanlong) [inline ceph_deploy/install.py]: @alfredodeza Ah, it should be **_s/repo/parser/_**  , updating...

**Comment by gaowanlong** [comment]
@alfredodeza What do you think about this change now? ;)

**Comment by alfredodeza** [comment]
This looks good. Before merging, could you follow the commit guidelines in https://github.com/ceph/ceph-deploy/blob/master/CONTRIBUTING.rst#commits ? It will require rewriting the commits with the tracker issue ID

**Comment by gaowanlong** [comment]
@alfredodeza yeah, updated with an issue tracker ID now. ;)

**Comment by vasukulkarni** [comment]
This is breaking tests, please revert this for now and should be tested for basic sanity for merge
http://qa-proxy.ceph.com/teuthology/vasu-2016-06-29_11:55:36-smoke:systemd-jewel---basic-vps/283750/teuthology.log

2016-06-29T12:44:45.450 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy.install][DEBUG ] Detecting platform for host vpm163 ...
2016-06-29T12:44:45.489 INFO:teuthology.orchestra.run.vpm163.stderr:[vpm163][DEBUG ] connection detected need for sudo
2016-06-29T12:44:45.516 INFO:teuthology.orchestra.run.vpm163.stderr:[vpm163][DEBUG ] connected to host: vpm163
2016-06-29T12:44:45.516 INFO:teuthology.orchestra.run.vpm163.stderr:[vpm163][DEBUG ] detect platform information from remote host
2016-06-29T12:44:45.541 INFO:teuthology.orchestra.run.vpm163.stderr:[vpm163][DEBUG ] detect machine type
2016-06-29T12:44:45.546 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy.install][INFO  ] Distro info: CentOS Linux 7.2.1511 Core
2016-06-29T12:44:45.546 INFO:teuthology.orchestra.run.vpm163.stderr:[vpm163][INFO  ] installing Ceph on vpm163
2016-06-29T12:44:45.546 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ] Traceback (most recent call last):
2016-06-29T12:44:45.547 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]   File "/home/ubuntu/cephtest/ceph-deploy/ceph_deploy/util/decorators.py", line 69, in newfunc
2016-06-29T12:44:45.547 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]     return f(*a, **kw)
2016-06-29T12:44:45.547 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]   File "/home/ubuntu/cephtest/ceph-deploy/ceph_deploy/cli.py", line 170, in _main
2016-06-29T12:44:45.547 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]     return args.func(args)
2016-06-29T12:44:45.547 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]   File "/home/ubuntu/cephtest/ceph-deploy/ceph_deploy/install.py", line 192, in install
2016-06-29T12:44:45.547 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]     gpgcheck = gpgcheck,
2016-06-29T12:44:45.548 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ] UnboundLocalError: local variable 'gpgcheck' referenced before assignment
2016-06-29T12:44:45.548 INFO:teuthology.orchestra.run.vpm163.stderr:[ceph_deploy][ERROR ]
2016-06-29T12:44:45.569 INFO:tasks.ceph_deploy:Error encountered, logging exception before tearing down ceph-deploy
2016-06-29T12:44:45.569 INFO:tasks.ceph_deploy:Traceback (most recent call last):
  File "/var/lib/teuthworker/src/ceph-qa-suite_wip-systemd-tests/tasks/ceph_deploy.py", line 246, in build_ceph_cluster

**Pull request merged**